website/src/components/subscriptions/overview/SubscriptionDisplay.tsx

```diff
@@ -0,0 +1,203 @@
+import type { ReactNode } from 'react';
+import type {
+    CountTrigger,
+    EvaluationInterval,
+    LapisFilter,
+    ProportionTrigger,
+    Subscription,
+} from '../../../types/Subscription.ts';
+import { type DateWindow, dateWindowConfig } from '../../../types/DateWindow.ts';
+import type { NotificationChannels } from '../../../types/NotificationChannels.ts';
+
+export function SubscriptionContent({ children, className = '' }: { children: ReactNode; className?: string }) {
+    return <div className={`rounded-xl border-2 border-gray-100 p-6 ${className}`}>{children}</div>;
+}
+
+export function SubscriptionDisplay({ subscription }: { subscription: Subscription }) {
+    return (
+        <>
+            <div className={'grid grid-cols-1 gap-2 sm:grid-cols-2 md:grid-cols-3'}>
+                <StatusDisplay
+                    active={subscription.active}
+                    triggerEvaluationResult={subscription.triggerEvaluationResult}
+                />
+                <EvaluationIntervalDisplay interval={subscription.interval} />
+                <DateWindowDisplay dateWindow={subscription.dateWindow} />
+                <TriggerDisplay trigger={subscription.trigger} />
+                <NotificationChannelDisplay notificationChannels={subscription.notificationChannels} />
+            </div>
+        </>
+    );
+}
+
+function StatusDisplay({
+    active,
+    triggerEvaluationResult,
+}: {
+    active: boolean;
+    triggerEvaluationResult: Subscription['triggerEvaluationResult'];
+}) {
+    return (
+        <SubscriptionContent className={'flex flex-col gap-2'}>
+            <div className={'font-bold text-gray-500'}>Status</div>
+            {!active && <div className={'text-gray-500'}>Subscription inactive. Activate to get notifications.</div>}
+
+            {active && <TriggerEvaluationResult result={triggerEvaluationResult} />}
```

Comment:
```suggestion
            {active ? (
                <TriggerEvaluationResult result={triggerEvaluationResult} />
            ) : (
                <div className={'text-gray-500'}>Subscription inactive. Activate to get notifications.</div>
            )}
```
I like this version better, because it reflects the "either this or that, but not both at the same time" aspect better.